shared/src/simulation/activities/load-vehicle.ts

```diff
@@ -212,6 +233,24 @@ export const loadVehicleActivity: SimulationActivity<LoadVehicleActivityState> =
                     );
                 });
 
+                const patientMovementsCount =
+                    patientsToUnload.length + patientsToLoad.length;
+
+                // Personnel has to leave and reenter the vehicle if patients are unloaded or loaded
+                const personnelLoadingRequired =
+                    personnelToLoadCount > 0 || patientMovementsCount > 0;
+
+                // Calculate loading time based on the patients and personnel to be loaded
+                // Do not do the calculation if the time is already set (which could occur if an instance of this activity was imported from an older state version)
+                if (activityState.loadDelay === 0) {
```

Comment:
Please use `undefined` as the initial value for loadDelay, if it should be calculated by the activity

Reply:
Changed it in https://github.com/hpi-sam/digital-fuesim-manv/pull/940/commits/8f74083abec6e9147e4af8edfd17dbeebdf98cb1 and https://github.com/hpi-sam/digital-fuesim-manv/pull/940/commits/07f036cae88272750488db19a0aaa07bc14f38f8